src/Home/Home.spec.tsx

```diff
@@ -1,101 +1,255 @@
 
-    it("it renders a notification are not enabled message", () => {
-      const enPermissionStatus: ENPermissionStatus = ["AUTHORIZED", "DISABLED"]
-      const requestPermission = jest.fn()
+      const enPermissionStatus: ENPermissionStatus = ["AUTHORIZED", "ENABLED"]
+      const permissionProviderValue = createPermissionProviderValue(
+        enPermissionStatus,
+      )
 
       const { getByTestId } = render(
-        <Home
-          enPermissionStatus={enPermissionStatus}
-          requestPermission={requestPermission}
-        />,
+        <PermissionsContext.Provider value={permissionProviderValue}>
+          <Home />
+        </PermissionsContext.Provider>,
       )
 
       const header = getByTestId("home-header")
       const subheader = getByTestId("home-subheader")
-
-      expect(header).toHaveTextContent("Exposure Notifications Disabled")
-      expect(subheader).toHaveTextContent(
-        "Enable Exposure Notifications to receive information about possible exposures",
+      const bluetoothStatusContainer = getByTestId(
+        "home-bluetooth-status-container",
+      )
+      const bluetoothDisabledText = within(bluetoothStatusContainer).getByText(
+        "Disabled",
       )
+
+      await wait(() => {
+        expect(header).toHaveTextContent("Inactive")
+        expect(subheader).toHaveTextContent(
+          "Enable Bluetooth and Proximity Tracing to get info about possible exposures",
+        )
+        expect(bluetoothDisabledText).toBeDefined()
+      })
     })
+  })
 
-    it("it renders an Enable Notifications button which requests permissions", async () => {
-      const enPermissionStatus: ENPermissionStatus = ["AUTHORIZED", "DISABLED"]
-      const requestPermission = jest.fn()
+  describe("When Bluetooth is disabled", () => {
+    it("it prompts the user to enable Bluetooth", async () => {
+      expect.assertions(1)
+      ;(isBluetoothEnabled as jest.Mock).mockResolvedValueOnce(false)
+
+      const enPermissionStatus: ENPermissionStatus = ["AUTHORIZED", "ENABLED"]
+      const permissionProviderValue = createPermissionProviderValue(
+        enPermissionStatus,
+      )
 
       const { getByTestId } = render(
-        <Home
-          enPermissionStatus={enPermissionStatus}
-          requestPermission={requestPermission}
-        />,
+        <PermissionsContext.Provider value={permissionProviderValue}>
+          <Home />
+        </PermissionsContext.Provider>,
       )
-      const button = getByTestId("home-request-permissions-button")
 
-      fireEvent.press(button)
-      await wait(() => {
-        expect(requestPermission).toHaveBeenCalled()
+      const fixBluetoothButton = getByTestId("home-bluetooth-status-container")
+      const alert = jest.spyOn(Alert, "alert")
+
+      await wait(async () => {
+        fireEvent.press(fixBluetoothButton)
+        await wait(() => {
+          expect(alert).toHaveBeenCalled()
```

Comment:
Since we have a couple of alert calls on the component how do you feel about adding arguments to this assertion?

Reply:
Sounds good!